docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3.7"
+services:
+ db:
+   image: postgres
+   environment:
+      DB_NAME: postgres
+      DB_USER: postgres
+      DB_PASS: password 
+   volumes:
+     - ./pgdata:/var/lib/postgresql/data
+   ports:
+     - '5432:5432'
+ web:
+   image: jordybastien/technites
```

Comment:
I think you don't have to put your image here, just  build it because we have Dockerfile in root directory

Reply:
makes sense, let me do it

Reply:
@mucyomiller You can check again